### PR TITLE
Make "EA640" an allowable `sonar_model` input to `open_raw`

### PR DESCRIPTION
**Pull request by leewujung**
This PR makes it possible to use `sonar_model="EA640"` in `open_raw`. The docstring is also updated.

This PR addresses #538.

### CODE REVIEW

File: echopype/convert/api.py [hunk 1]
```diff
@@ -338,8 +338,8 @@ def open_raw(
 
         - ``EK60``: Kongsberg Simrad EK60 echosounder
         - ``ES70``: Kongsberg Simrad ES70 echosounder
-        - ``EK80``: Kongsberg Simrad EK80 and Kongsberg EA640 echsoounders
-        - ``EK80``: Kongsberg Simrad EK80 and Kongsberg EA640 echsoounders
+        - ``EK80``: Kongsberg Simrad EK80 echsoounder
```
Comment (emiliom) [inline echopype/convert/api.py:341]: Spelling mistake, echsoounder

File: echopype/convert/api.py [hunk 2]
```diff
@@ -338,8 +338,8 @@ def open_raw(
...
+        - ``EA640``: Kongsberg EA640 echsoounder
```
Comment (emiliom) [inline echopype/convert/api.py:342]: Spelling mistake, echsoounder

File: echopype/convert/set_groups_ek80.py [hunk 3]
```diff
@@ -24,20 +24,13 @@ def set_env(self, env_only=False) -> xr.Dataset:
...
-                    ["ping_time"],
-                    [self.parser_obj.environment["temperature"]],
-                ),
-                "depth": (["ping_time"], [self.parser_obj.environment["depth"]]),
-                "acidity": (["ping_time"], [self.parser_obj.environment["acidity"]]),
-                "salinity": (["ping_time"], [self.parser_obj.environment["salinity"]]),
-                "sound_speed_indicative": (
-                    ["ping_time"],
-                    [self.parser_obj.environment["sound_speed"]],
-                ),
```
Comment (emiliom) [inline echopype/convert/set_groups_ek80.py:39]: Note that your new, more compact scheme doesn't work for sound speed, b/c the input and output variable names are different: `sound_speed` vs `sound_speed_indicative`.  That is, unless the original code was wrong; but in that case I suggest you make a note that you're also correcting an error.

File: echopype/convert/api.py (hunk 2 above)
Comment (leewujung) [reply echopype/convert/api.py:342]: Hah, thanks for catching this, I guess I wasn't actually reading! :D

File: echopype/convert/set_groups_ek80.py (hunk 3 above)
Comment (leewujung) [reply echopype/convert/set_groups_ek80.py:39]: This with the last one clearly showed that I wasn't awake... -_-

It should be `sound_speed_indicative` as defined in the convention. If there's absorption it should be `absorption_indicative`. I'll change that.

Comment (leewujung) [reply echopype/convert/set_groups_ek80.py:39]: I'll include this as part of #537.

File: echopype/convert/api.py (hunk 2 above)
Comment (leewujung) [reply echopype/convert/api.py]: Done!

**Pull request merged**